disco/types/user.py

```diff
@@ -16,14 +16,36 @@ class DefaultAvatars(object):
     ALL = [BLURPLE, GREY, GREEN, ORANGE, RED]
 
 
+class UserFlags(object):
+    NONE = 0
+    DISCORD_EMPLOYEE = 1 << 0
+    DISCORD_PARTNER = 1 << 1
+    HS_EVENTS = 1 << 2
+    BUG_HUNTER = 1 << 3
+    HS_BRAVERY = 1 << 6
+    HS_BRILLIANCE = 1 << 7
+    HS_BALANCE = 1 << 8
+    EARLY_SUPPORTER = 1 << 9
+    TEAM_USER = 1 << 10
+
+
+class PremiumType(object):
+    CLASSIC = 1
+    NITRO = 2
+
+
 class User(SlottedModel, with_equality('id'), with_hash('id')):
     id = Field(snowflake)
     username = Field(text)
-    avatar = Field(text)
     discriminator = Field(text)
+    avatar = Field(text)
     bot = Field(bool, default=False)
+    mfa_enabled = Field(bool)
+    locale = Field(text)
     verified = Field(bool)
     email = Field(text)
+    flags = Field(int)
+    premium_type = Field(int)
```

Comment:
The following are only granted after OAuth. Unsure if they should be in here just for the sake of having them or not.
- flags
- premium_type 
- locale
- mfa_enabled

Reply:
Only added because `email` and `verified` are the same kind of values, and assuming that if they're there for a reason, then these may also be useful, otherwise all six can be removed. 

Will let b1nzy decide.